### PR TITLE
Build tikv-importer as a standalone binary

### PR DESCRIPTION
**Pull request by huachaohuang**
To reuse some common functions in `tikv-server.rs`, I move them to `tikv_setup.rs`.
And add `tikv-importer.toml` as a recommend configuration template.

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -38,27 +38,27 @@ extern crate signal;
 extern crate tikv;
 extern crate toml;
 
-mod signal_handler;
 #[cfg(unix)]
 mod profiling;
+#[macro_use]
+mod tikv_setup;
```
Comment (overvenus) [inline src/bin/tikv-server.rs]: Why not remove the prefix?

Comment (huachaohuang) [reply src/bin/tikv-server.rs]: The prefix looks fine to me, why do you want to remove it?

Comment (overvenus) [reply src/bin/tikv-server.rs]: It looks like a standalone binary.

**Pull request merged**